ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -285,6 +285,8 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
             auto const& N = ip_data.N;
             auto const& dNdx = ip_data.dNdx;
 
+            pos.setIntegrationPoint(ip);
```

Comment:
Good observation!

Reply:
Maybe extraction to separate PR?

Reply:
After discussed with Tom, we keep it as it is in this PR.